### PR TITLE
Backend eps

### PR DESCRIPTION
**Pull request by mganahl**

### CODE REVIEW

**Comment by mganahl** [comment]
@alewis, review appreciated!

File: tensornetwork/backends/jax/jax_backend_test.py [hunk 1]
```diff
@@ -1254,4 +1254,8 @@ def test_power(dtype):
   actual = backend.power(base_tensor, power)
   expected = jax.numpy.power(base_tensor, power)
   np.testing.assert_allclose(expected, actual)
-  
+
+@pytest.mark.parametrize("dtype", np_dtypes)
+def test_eps(dtype):
+  backend = jax_backend.JaxBackend()
+  assert backend.eps(dtype) == np.finfo(dtype).eps
```
Comment (jacksonwb) [inline tensornetwork/backends/jax/jax_backend_test.py:1261]: is this supposed to be ` == jnp.finfo(dtype).eps` ?

Comment (mganahl) [reply tensornetwork/backends/jax/jax_backend_test.py:1261]: Yes, to be strictly speaking. I use jnp and np interchangeably for these kind of tests

**Comment by jacksonwb** [comment]
LTM

**Review by jacksonwb — APPROVED** [review]
LGTM

**Pull request merged**